web/src/components/app-survey/SurveyCard.tsx

```diff
@@ -0,0 +1,53 @@
+import { useTranslation } from 'react-i18next';
+
+import FeedbackCard, { SurveyResponseProps } from './FeedbackCard';
+
+async function postSurveyResponse({
+  feedbackScore,
+  inputText,
+  reference: surveyReference,
+  location = 'Unknown',
+}: SurveyResponseProps) {
+  try {
+    const response = await fetch(
+      `https://hooks.zapier.com/hooks/catch/14671709/3vzj7zn/`,
+      {
+        method: 'POST',
+        body: JSON.stringify({
+          score: feedbackScore,
+          feedback: inputText,
+          reference: surveyReference,
+          location,
+        }),
+      }
+    );
+
+    if (!response.ok) {
+      throw new Error('Network response was not ok');
+    }
+
+    const result = await response.json();
+    if (result.result === 'success') {
+      console.log('Feedback submitted successfully.');
```

Comment:
Should we remove this one, or at least make it console.debug?

Reply:
I had some un-pushed changes removing this, should be updated now :)